docs/source/conf.py

```diff
@@ -68,18 +68,24 @@ class CustomGalleryExampleSortKey:
     def __init__(self, src_dir):
         self.src_dir = src_dir
 
-    order = [
-        "basic_example.py",
-        "audio_decoding.py",
-        "basic_cuda_example.py",
-        "file_like.py",
-        "approximate_mode.py",
-        "sampling.py",
-    ]
-
     def __call__(self, filename):
+        if "examples/decoding" in self.src_dir:
+            order = [
+                "basic_example.py",
+                "audio_decoding.py",
+                "basic_cuda_example.py",
+                "file_like.py",
+                "approximate_mode.py",
+                "sampling.py",
+            ]
+        else:
+            assert "examples/encoding" in self.src_dir
+            order = [
+                "audio_encoding.py",
+            ]
+
```

Comment:
Can we add a comment explaining that we have two top-level galleries, and for that reason, we need to figure out which gallery we're using (decoding versus encoding)? I was real confused until I concluded that must be what's going on.